src/main/kotlin/io/github/graphglue/authorization/MergedAuthorization.kt

```diff
@@ -1,20 +1,11 @@
 package io.github.graphglue.authorization
 
-import io.github.graphglue.definition.RelationshipDefinition
-import io.github.graphglue.model.Authorization
 import io.github.graphglue.model.Rule
 
-/**
```

Comment:
Why remove docs?

Reply:
MergedAuthorization has temporarily been removed, when it was readded, I forgot the docs. 
Readded with 93aa0fc9fcb486b06d6d455542ad56976607f170